Formula/osc.rb

```diff
@@ -35,6 +35,11 @@ class Osc < Formula
     sha256 "05d94fd9b2dae2fb8e072819a795f0e05d3611b09ea185f68e1630530ec09ae8"
   end
 
```

Comment:
If you install M2Crypto and run poet against it, you'll get the typing stanza, but getting either osc or M2Crypto installed is annoying. Do you remember getting output that lacked typing?